### ISSUE TITLE
Document if CAxxxx rule is enabled by default or not

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/docs#20796]
Currently the CAxxxx documentation doesn't include whether the rule is enabled by default or not.

As this changes between releases, the doc may mention the latest version or two.

Currently, we can just include .NET 5.0 only, and later on when .NET 6.0 is released, the doc should include both. Later when .NET 7.0 is released, drop .NET 5.0 and include only .NET 6.0 and .NET 7.0, etc.

If the default for rules change between minor versions, that would still be a problem.

cc: @mavasani @gewarren

**Comment by BillWagner** [comment]
related to #20560

**Comment by Evangelink** [comment]
Happy to help with the doc change if we agree upon a format/way to state changes depending on version.

**Comment by gewarren** [comment]
@Evangelink Thanks for that offer! I've made a draft PR [here](https://github.com/dotnet/docs/pull/28595). What do you think of adding the default severity to the table at the top of each rule's page?

![image](https://user-images.githubusercontent.com/24882762/157534375-3727fa5e-b14b-4490-a51e-105b2aca1d9b.png)

**Comment by Evangelink** [comment]
I think that's perfect for the severity. Maybe we could have a sub-section in case of a version change (fake example, let's say it's Warning in .NET 5 and Info in .NET 6), I'd be in favor of showing both not only latest.

In term of seeing differences by version, I guess that in a perfect world, I'd like to be able to see what/how things changed so that when I am visiting this page for an older version I could see what was the severity at this point. Maybe rather than trying to make too complex things, we should just try to always specify which version adds or change something rather than considering the doc to be a view over the most recent state.
At first I was thinking about having some drop down to select the version and show the page as-is from this "release" but that would be putting a strong limit on how much we can change the UI so I don't like that.

**Comment by gewarren** [comment]
> I think that's perfect for the severity. Maybe we could have a sub-section in case of a version change (fake example, let's say it's Warning in .NET 5 and Info in .NET 6), I'd be in favor of showing both not only latest.

Yes if the severity is different in different versions, then we should probably have two rows in the table to differentiate it.

**Comment by mavasani** [comment]
In an ideal world, we would be able to insert the [Analyzer releases file](https://github.com/dotnet/roslyn-analyzers/blob/main/src/NetAnalyzers/Core/AnalyzerReleases.Shipped.md) into the docs repo and some tooling will extract and auto-populate the severity. This file should also contain the severity change history, rule removal history, etc. with different .NET releases, so has everything that @Evangelink requested above. I am not sure how feasible it is tooling wise to have this automated.

**Comment by BillWagner** [comment]
Closing as completed
